pgjdbc/src/main/java/org/postgresql/jdbc/PgConnection.java

```diff
@@ -224,11 +222,6 @@ public PgConnection(HostSpec[] hostSpecs,
     // Now make the initial connection and set up local state
     this.queryExecutor = ConnectionFactory.openConnection(hostSpecs, user, database, info);
 
-    // WARNING for unsupported servers (8.1 and lower are not supported)
-    if (LOGGER.isLoggable(Level.WARNING) && !haveMinimumServerVersion(ServerVersion.v8_2)) {
```

Comment:
Please keep this warning and just update the ServerVersion to v9_0, this way "some" users will notice that this version of the driver "might" not work.